### DISCUSSION TITLE
Building for macos

### DISCUSSION THREAD

**Discussion by LETIshNick** [briandk/transcriptase#154] [Q&A]
Hi, 

I've been using transcriptase for the last two years during my Master's in sociology and I like it very much as well as my schoolmates. I relied on the latest available version from Mac App Store on my 2011 mbp with Catalina. But now, it's no longer available on Big Sur, at least for the M1 ones. So, I just copied the app from my old machine and it works flawlessly. 

Nevertheless I'd like to know is it possible to build the software for macos? Are there any instructions? Although I have experience in building software for unix-like systems that largely surpasses the simplistic 'make, make install', I completely lack knowledge of electron-based apps. Hence, I would greatly appreciate any indications. 

Thanks in advance.

**Comment by briandk** [comment]
Hi Nick,

I'm thrilled that Transcriptase has been a help to you.

Unfortunately, I had medical issues earlier this year which kept me from updating the app. As a result, I've fallen behind on upkeep when it comes to keeping the software dependencies [up to date](https://github.com/briandk/transcriptase/pulls).

The app should still work for the foreseeable future, but I can't guarantee for how long.

I'm really sorry about this.
